plugins/inputs/snmp/README.md

```diff
@@ -1,180 +1,221 @@
+      oid = "IF-MIB::ifDescr"
 
-* `oid_index_suffix`:
-The OID sub-identifier to strip off so that the index can be matched against other fields in the table.
+      ## Name of the field or tag to create.  If not specified, it defaults to
+      ## the value of 'oid'. If 'oid' is numeric an attempt to translate the
+      ## numeric OID into a textual OID will be made.
+      # name = ""
 
-* `oid_index_length`:
-Specifies the length of the index after the supplied table OID (in OID path segments). Truncates the index after this point to remove non-fixed value or length index suffixes.
+      ## Output this field as a tag.
+      # is_tag = false
 
-* `name`:
-Output field/tag name.
-If not specified, it defaults to the value of `oid`. If `oid` is numeric, an attempt to translate the numeric OID into a texual OID will be made.
+      ## The OID sub-identifier to strip off so that the index can be matched
+      ## against other fields in the table.
+      # oid_index_suffix = ""
 
-* `is_tag`:
-Output this field as a tag.
+      ## Specifies the length of the index after the supplied table OID (in OID
+      ## path segments). Truncates the index after this point to remove non-fixed
+      ## value or length index suffixes.
+      # oid_index_length = 0
+```
 
-* `conversion`: Values: `"float(X)"`,`"float"`,`"int"`,`""`. Default: `""`
-Converts the value according to the given specification.
+### Troubleshooting
 
-    - `float(X)`: Converts the input value into a float and divides by the Xth power of 10. Efficively just moves the decimal left X places. For example a value of `123` with `float(2)` will result in `1.23`.
-    - `float`: Converts the value into a float with no adjustment. Same as `float(0)`.
-    - `int`: Convertes the value into an integer.
-    - `hwaddr`: Converts the value to a MAC address.
-    - `ipaddr`: Converts the value to an IP address.
+Check that a numeric field can be translated to a textual field:
+```
+$ snmptranslate .1.3.6.1.2.1.1.3.0
+DISMAN-EVENT-MIB::sysUpTimeInstance
+```
 
-#### Table parameters:
-* `oid`:
-Automatically populates the table's fields using data from the MIB.
+Request a top-level field:
+```
+$ snmpget -v2c -c public 127.0.0.1 sysUpTime.0
+```
 
-* `name`:
-Output measurement name.
-If not specified, it defaults to the value of `oid`.  If `oid` is numeric, an attempt to translate the numeric OID into a texual OID will be made.
+Request a table:
+```
+$ snmptable -v2c -c public 127.0.0.1 ifTable
+```
 
-* `inherit_tags`:
-Which tags to inherit from the top-level config and to use in the output of this table's measurement.
+To collect a packet capture, run this command in the background while running
+Telegraf or one of the above commands.  Adjust the interface, host and port as
+needed:
+```
+$ sudo tcpdump -s 0 -i eth0 -w telegraf-snmp.pcap host 127.0.0.1 and port 161
+```
 
-* `index_as_tag`:
-Adds each row's index within the table as a tag.  
+### Example Output
 
-### MIB lookups
```

Comment:
The snmp trap input's readme refers to this section by name.  Since the section is removed in this change we'll need to update snmp_trap/README.md.  It looks like the content is moved to the prerequisites section.